src/fileutil.c

```diff
@@ -63,6 +63,15 @@ rs_file_open(char const *filename, char const *mode)
 	    return stdin;
     }
 
+    if (!force && is_write) {
+	if ((f = fopen(filename, "rb"))) {
```

Comment:
Personally I would just use open/fdopen unless you think that's going to be annoying on Windows?